src/model/ops.py

```diff
@@ -1,16 +1,317 @@
-"""
-    This script was made by Nick at 19/07/20.
-    To implement code of your operation to be being used your network.
-"""
+import math
+import re
+import sys
 
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchaudio
+from torch.autograd import Variable
 
-def multiply(a, b):
-    return a * b
 
+class Conv_2d(nn.Module):
+    def __init__(self, input_channels, output_channels, shape=3, stride=1, pooling=2):
+        super(Conv_2d, self).__init__()
+        self.conv = nn.Conv2d(
+            input_channels, output_channels, shape, stride=stride, padding=shape // 2
+        )
+        self.bn = nn.BatchNorm2d(output_channels)
+        self.relu = nn.ReLU()
+        self.mp = nn.MaxPool2d(pooling)
 
-def add(a, b):
-    return a + b
+    def forward(self, x):
+        out = self.mp(self.relu(self.bn(self.conv(x))))
+        return out
 
 
-def subtract(a, b):
-    return a - b
+class Res_2d_mp(nn.Module):
```

Comment:
python의 클래스 컨벤션은 CamelCase로 쓰는게 좋을 것 같습니다.

```suggestion
class Res2Dmp(nn.Module):
```